crates/ln-dlc-node/src/tests/dlc_channel.rs

```diff
@@ -249,38 +251,126 @@ async fn can_open_and_force_close_channel() {
     // or similar
 }
 
-async fn setup_channel_with_position() -> (
-    Arc<Node<TenTenOneInMemoryStorage, InMemoryStore>>,
+#[tokio::test(flavor = "multi_thread")]
+#[ignore]
+async fn can_open_channel_with_min_inputs() {
+    init_tracing();
+
+    let app_dlc_collateral = Amount::from_sat(10_000);
+    let coordinator_dlc_collateral = Amount::from_sat(10_000);
+
+    // We must fix the fee rate so that we can predict how many sats `rust-dlc` will allocate
+    // for transaction fees.
+    let fee_rate_sats_per_vbyte = 2;
+    let expected_fund_tx_fee = 252 * fee_rate_sats_per_vbyte;
+
+    // This also depends on the fee rate, but the formula is a bit more involved.
+    let fee_reserve = 880;
```

Comment:
I can work on using constants defined by `rust-dlc`, to make all this clearer and less flaky on `rust-dlc` upgrades. But I think this suffices for now.